view/gui.py

```diff
@@ -0,0 +1,46 @@
+import streamlit as st
+import folium
+import osmnx
+import networkx as nx
+import leafmap.foliumap as leafmap
+import json
+
+class gui:
+    def __init__(self, filename = "config.json"):
+        with open(filename) as config_file:
+            self.config = json.load(config_file)
+    
+    def clear_text(self):
+        st.session_state["go_from"] = ""
+        st.session_state["go_to"] = ""
+
+    def generate_and_show_gui(self):
+        with st.sidebar:
+            st.markdown("<h1 style='text-align: center;font-size:100px;'>🧭</h1>", unsafe_allow_html=True)
+            st.markdown("<h1 style='text-align: center;'>EleNa: Elevation Based Navigation System</h1>", unsafe_allow_html=True)
+
+            BASEMAPS = self.config['BASEMAPS']
+            TRAVEL_MODE = self.config['TRAVEL_MODE']
+            ADDRESS_DEFAULT = self.config['ADDRESS_DEFAULT']
+
+            basemap = st.selectbox("Select Map Type", BASEMAPS)
```

Comment:
Can be changed to `basemap = st.selectbox("Select Map Type", self.config["BASEMAPS"])`. Same for other attributes mentioned in the config.

Reply:
Done.